primitives/primproc/columncommand.cpp

```diff
@@ -282,11 +282,24 @@ void ColumnCommand::issuePrimitive()
         //if (wasVersioned && outMsg->ValidMinMax)
         //	cout << "CC: versioning overriding min max data\n";
         bpp->lbidForCP = lbid;
-        if (primMsg->DataSize > 8)
+        if (UNLIKELY(utils::isWide(primMsg->DataSize)))
         {
-            bpp->hasBinaryColumn = true;
-            bpp->bigMaxVal = outMsg->Max;
-            bpp->bigMinVal = outMsg->Min;
+            if (datatypes::Decimal::isWideDecimalType(colType))
```

Comment:
In the outer if, we are checking for primMsg->DataSize. While in the inner if, we are using colType.colWidth. Can they be unequal?

Reply:
As of now no, b/c colType is ctored inside ::prep and colType.colWidth is taken from primMsg->DataSize.

Reply:
But I will make it consistent.